### PR TITLE
Prevent NullPointerException when unsubscribe categoris

### PR DESCRIPTION
**Pull request by yaniv-cb4**
NullPointerException is raised when you hit enable/disable for invalidate tokens.

### CODE REVIEW

File: service/src/main/java/org/jboss/aerogear/unifiedpush/service/util/GCMTopicManager.java [hunk 1]
```diff
@@ -68,6 +68,10 @@ public GCMTopicManager(AndroidVariant variant) {
             return new HashSet<String>(0);
         }
         JSONObject rel = (JSONObject) info.get("rel");
+        if (rel == null){
```
Comment (matzew) [inline service/src/main/java/org/jboss/aerogear/unifiedpush/service/util/GCMTopicManager.java]: @matskiv can you take a look ?

**Comment by matzew** [comment]
SO, using the FCM enabled SDKs, I don't really see the mentioned NPE here.

However, I am fine with merging, this extra check, @yaniv-cb4

File: service/src/main/java/org/jboss/aerogear/unifiedpush/service/util/GCMTopicManager.java [hunk 2]
```diff
@@ -68,6 +68,10 @@ public GCMTopicManager(AndroidVariant variant) {
...
+        	logger.debug("Couldn't parse rel object Instance ID service.");
+            return new HashSet<String>(0);
```
Comment (idelpivnitskiy) [inline service/src/main/java/org/jboss/aerogear/unifiedpush/service/util/GCMTopicManager.java]: Collections.emptySet()?

**Comment by matzew** [comment]
@yaniv-cb4 thansk a lot, 👍 

I will port this to master tomorrow

**Pull request merged**